Mbonigabay/Free-Mentor-2
# #168367145 sign in with postgres

#### What does this PR do?

- Build a sign in with postgres

#### Description of Task to be completed?

- N/A

#### How should this be manually tested?

- Created a sign up endpoint

- Add a middleware to verify token

- Test the endpoint

#### Any background context you want to provide?

- N/A

#### What are the relevant pivotal tracker stories?

- [https://www.pivotaltracker.com/story/show/168367145](https://www.pivotaltracker.com/story/show/168367145)

#### Screenshots (if appropriate)
![SigninReq](https://user-images.githubusercontent.com/47386346/64681766-9ad40780-d480-11e9-8705-b77a60c2b9f1.PNG)
![SigninRes](https://user-images.githubusercontent.com/47386346/64681764-9ad40780-d480-11e9-9f43-dec0d4e8af66.PNG)


#### Questions:

- N/A
